contribute/dev_setup.rst

```diff
@@ -0,0 +1,100 @@
+=================
+Development Setup
+=================
+
+This section describes how someone wanting to contribute to the OpenCMISS project should set up their *working copy* for developing OpenCMISS.
+
+.. contents::
+
+Overview
+========
+
+The OpenCMISS codebase is hosted on GitHub and therefore Git is used to track changes in the codebase.  Before you begin you will need to have a few prerequisites satisfied:
+
+#. GitHub user account (for the rest of this document we will call our user *Prasad*)
+#. Git installed
+
+This document doesn't cover the process of satisfying these prerequisites, it is left as an exercise for you (Google is your friend).  The remainder of this document assumes that these prerequisites have been met.  This document covers setup from the command line if you are using a GUI like 'GitHub for Windows' or 'Pycharm' then you will need to adjust the commands for the GUI you are using.
+
+The goal here is to get a working copy of source code, tests, and documentation onto your PC so that you can begin development.  To make this happen you will need to fork a :term:`prime OpenCMISS repository`, make a clone onto your PC, and set up the Git remotes.  In :numref:`fig_devSetup_githubRepos` you can see a pictorial representation of what we are aiming to achieve.
+
+.. _fig_devSetup_githubRepos:
+
+.. figure:: images/OpenCMISSProcesses-GitHubRepos.png
+   :align: center
+   :alt: Setup of Git repositories
+
+   Setup of repositories for development
+
+The four steps to getting set up are detailed below.
+
+Forking your own copy
+=====================
+
+Login to GitHub using your credentials and go to the OpenCMISS GitHub organisation page::
+
+   https://github.com/OpenCMISS
+
+The OpenCMISS organisation contains the git repositories for each component of OpenCMISS:
+
+   - https://github.com/OpenCMISS/iron
+   - https://github.com/OpenCMISS/zinc
+   - https://github.com/OpenCMISS/examples
+   - https://github.com/OpenCMISS/documentation
+   - https://github.com/OpenCMISS/website
+   - https://github.com/OpenCMISS/manage
+
+Select the repository that you wish to develop and use the fork button to create a fork of the repository under your own account, see :numref:`fig_devSetup_githubFork` for locating this button. The remainder of this documentation will assume you have selected the iron repository, however, the steps outlined below can be applied to any of the repositories in the OpenCMISS organisation.
+
+.. _fig_devSetup_githubFork:
+
+.. figure:: images/OpenCMISSProcesses-GitHubForkButton.png
+   :align: center
+   :alt: Fork button of the OpenCMISS iron repository
+
+   Fork button for the OpenCMISS iron repository
+
+Clone
+=====
+
+You now need to clone the selected repository to your PC.  Do this by going to your fork (in this example user *prasad*'s fork) at::
+
+   https://github.com/prasad/iron
```

Comment:
I really don't like having an actual person's name here; keep credit in the git contribution record or a dedicated contributors page. Change 'prasad' to 'username' (which is what github calls it), and at the beginning define that it is to be substituted with your own username on github. You could alternatively write <username> or emphasise it with italics.